finder-app/finder_bk.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
```

Comment:
@Sharath-Jonnala use revision control to save old versions of files rather than creating unused files in your repo.